File: Quiz.Entity/Models/Carousel.cs
using System.ComponentModel.DataAnnotations;

namespace Quiz.Entity.Models
{
    public partial class Carousel
    {
        [Key]
        public int CarouselId { get; set; }
        public string Title { get; set; }
        public string Image { get; set; }
        public bool IsActive { get; set; }
        public int AppId { get; set; }
        public int SerialNo { get; set; }
    }
}

File: Quiz.Android/lib/models/carousel.dart
class Carousel {
  late int carouselId;
  late String title;
  late String image;
  late bool isActive;
  late int serialNo;

  Carousel({
    required this.carouselId,
    required this.title,
    required this.image,
    required this.isActive,
    required this.serialNo,
  });

  factory Carousel.fromJson(Map<String, dynamic> json) {
    return Carousel(
      carouselId: json["carouselId"],
      title: json["title"],
      image: json["image"],
      isActive: json["isActive"],
      serialNo: json["serialNo"],
    );
  }
}
